spec/models/like_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before :all do
+    @author = User.create(name: 'Okari Nyandika', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                          bio: 'Student a microverse.')
+    @user = User.create(name: 'Samsom Opiyo', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Teacher from Poland.')
+    @post = Post.create(author: @author, title: 'Okari Nyandika', text: 'Hello World', comments_counter: 0,
+                        likes_counter: 0)
+  end
+
+  context '#create' do
+    it 'is valid with existing user and post' do
+      expect(Like.new(user: @user, post: @post)).to be_valid
+    end
+
+    it 'is not valid without post' do
+      expect(Like.new(user: @user)).to_not be_valid
+    end
+
+    it 'is not valid without user' do
+      expect(Like.new(post: @post)).to_not be_valid
+    end
+  end
+
+  context '#update_likes_counter' do
```

Comment:
- [ ] Please make sure to test all your methods in your User model for example the `update_likes_counter` method.

Reply:
I don't have that method in my User model.

Reply:
Hello @2004-okari , I meant to say the Like model. That was a typo on my part. Sorry about that. These are the methods seen in your Like model:
![image](https://github.com/2004-okari/BLOG_APP/assets/45460192/fe442c06-3ba4-421c-aff8-87ef27f403f8)

Reply:
hello @Lucas-Masaba ,I do have tests for that method
```
context '#update_likes_counter' do
    before :all do
      5.times { Like.create(user: @user, post: @post) }
    end

    it 'keeps track of likes and equals 5' do
      expect(@post.likes_counter).to eq 5
    end
  end
```